cmd/devtool/devtool.go

```diff
@@ -20,27 +22,39 @@ import (
 )
 
 const (
-	gethMiningAccount = "87da6a8c6e9eff15d703fc2773e32f6af8dbe301"
-	controllerAddr    = "93ad00a63b14492386df9f1cc123d785705bdf99"
-	clientIdentifier  = "geth" // Client identifier to advertise over the network
-	passphrase        = ""
-	serviceURI        = "https://127.0.0.1:8936"
+	clientIdentifier = "geth" // Client identifier to advertise over the network
+	passphrase       = ""
+	serviceURI       = "https://127.0.0.1:8936"
 )
 
 var (
-	ethTxTimeout = 600 * time.Second
-	endpoint     = "ws://localhost:8546/"
+	ethTxTimeout              = 600 * time.Second
+	endpoint                  = "ws://localhost:8546/"
+	gethMiningAccount         = "87da6a8c6e9eff15d703fc2773e32f6af8dbe301"
+	gethMiningAccountOverride = false
+	controllerAddr            = "0x04B9De88c81cda06165CF65a908e5f1EFBB9493B"
```

Comment:
What are some example situations in which you'd have to change the `gethMiningAccount` and/or `controllerAddr`, and set the `gethMiningAccountOverride` and `controllerAddrOverride` to true?

Reply:
it is for the cases I'm not accounted for - for example, probably it will not work on Rinkeby, because I'm trying to enumerate events from first block till latest.